sdcm/sct_config.py

```diff
@@ -1516,6 +1516,13 @@ class SCTConfiguration(dict):
 
         dict(name="validate_large_collections", env="SCT_VALIDATE_LARGE_COLLECTIONS", type=boolean,
```

Comment:
Probably also should be added check for backend. No?

Reply:
what do you mean ? validation for `kafka_backend` and needed parameters ?

Reply:
2 parts:
- on which cluster backend it can run
- kafka backend configuration correctness/validation

Reply:
for the first there's no current limitation on backend, in theory it can run with any backend

as for kafka_backend, only one is implemented, validation for it is pointless at this point.

for the 2nd there is validation